webauthn/securecontext.http.html

```diff
@@ -17,7 +17,7 @@
     // Example 1
     // http://example.com/ opened in a top-level browsing context is not a secure context, as it was not delivered over an authenticated and encrypted channel.
     test (() => {
-        assert_false (typeof navigator.credentials.create === "function");
+        assert_false (typeof navigator.credentials === "object" && typeof navigator.credentials.create === "function");
```

Comment:
Note: This throws on line helpers.js:426 in `ensureInterface` because `navigator.credentials` doesn't exist, so `navigator.credentials.create` throws with an uncaught TypeError.

Perhaps that could be a separate commit?

Reply:
I had fixed i in one file but not the other... should be right in a0a0e8c

Reply:
Sorry, the issue is in `helpers.js` line 450 (now):

https://github.com/w3c/web-platform-tests/blob/4d8efcb6b6f4b784989c23cf83f46255388871a3/webauthn/helpers.js#L372-L374